templates/ethernet_RedHat.j2

```diff
@@ -3,6 +3,9 @@
 DEVICE={{ item.device }}
 {% if item.bootproto == 'static' %}
 BOOTPROTO=none
+{% if item.hwaddr is defined %}
+HWADDR={{ item.hwaddr }}
```

Comment:
Could you also support this for bonds and bridges? Also for Debian/Ubuntu?

Reply:
And please update the README with an example.

Reply:
I'll look at possibilities for bonds and bridges & Debian-like distros.
I'll add info to the readme also

Reply:
Do you want to set HWADDR for master or slave interfaces? I suggest you meant the master interface.

Reply:
Master

Reply:
I added support for bonds/bridges for both RedHat and Debian. Readme updated also.